integration-tests/src/test/resources/bash-scripts/shutdown-remoteconsole.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Can u hard-code the script in TestAction.shutdownWlsRemoteConsole() as a string parameter

Reply:
Done.